src/main/java/uk/gov/hmcts/reform/sendletter/services/ftp/FtpClient.java

```diff
@@ -52,9 +63,13 @@ public void upload(LocalSourceFile file, boolean isSmokeTestFile) {
                 String path = String.join("/", folder, file.getName());
                 sftp.getFileTransfer().upload(file, path);
 
+                success = true;
```

Comment:
I hadn't spotted that we tracked success! Happy with either approach.

Reply:
yea, only two key features of dependency: success and time took